addon/components/ember-popper-base.js

```diff
@@ -81,85 +92,90 @@ export default Component.extend({
    * @argument({ defaultIfUndefined: true })
    * @type('boolean')
    */
-  renderInPlace: false,
+  get renderInPlace() {
+    return this.args.renderInPlace ?? false;
+  }
 
   // ================== PRIVATE PROPERTIES ==================
 
   /**
    * Tracks current/previous state of `_renderInPlace`.
    */
-  _didRenderInPlace: false,
+  @tracked _didRenderInPlace = false;
 
   /**
    * Tracks current/previous value of `eventsEnabled` option
    */
-  _eventsEnabled: null,
+  @tracked _eventsEnabled = null;
 
   /**
    * Parent of the element on didInsertElement, before it may have been moved
    */
-  _initialParentNode: null,
+  @tracked _initialParentNode = null;
 
   /**
    * Tracks current/previous value of `modifiers` option
    */
-  _modifiers: null,
+  @tracked _modifiers = null;
 
   /**
    * Tracks current/previous value of `onCreate` callback
    */
-  _onCreate: null,
+  @tracked _onCreate = null;
 
   /**
    * Tracks current/previous value of `onUpdate` callback
    */
-  _onUpdate: null,
+  @tracked _onUpdate = null;
 
   /**
    * Tracks current/previous value of `placement` option
    */
-  _placement: null,
+  @tracked _placement = null;
 
   /**
    * Set in didInsertElement() once the Popper is initialized.
    * Passed to consumers via a named yield.
    */
-  _popper: null,
+  @tracked _popper = null;
 
   /**
    * Tracks popper element
    */
-  _popperElement: null,
+  @tracked _popperElement = null;
 
   /**
    * Tracks current/previous value of popper target
    */
-  _popperTarget: null,
+  @tracked _popperTarget = null;
 
   /**
    * Public API of the popper sent to external components in `registerAPI`
    */
-  _publicAPI: null,
+  @tracked _publicAPI = null;
 
   /**
    * ID for the requestAnimationFrame used for updates, used to cancel
    * the RAF on component destruction
    */
-  _updateRAF: null,
+  @tracked _updateRAF = null;
 
-  // ================== LIFECYCLE HOOKS ==================
+  constructor() {
+    super(...arguments);
 
-  willDestroyElement() {
-    this._super(...arguments);
-    if (this._popper !== null) {
-      this._popper.destroy();
-    }
-    raf.forget(this._updateRAF);
-  },
+    registerDestructor(this, () => {
```

Comment:
Tu dois pouvoir utiliser le hook willDestroy https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_lifecycle-and-properties